### DISCUSSION TITLE
Table Filter Stopped Working Following PHP 7x Upgrade

### DISCUSSION THREAD

**Discussion by bcr123psu** [koalyptus/TableFilter#818] [Q&A]
I'm hoping someone can jump into the wayback machine.  I set up this table filter back in 2012 or 2013 and everything was going fine until the fall when I had to upgrade my site to PHP 7x.  Suddenly, the table filter stopped working all together.  I haven't upgraded any aspect of this since it was originally put into production 8 years ago so I'm curious whether anything with PHP 7 might be impacting the filter or if there are any other thoughts on what the culprit might be.  Thanks in advance!

**Comment by cjohnsonuk** [comment]
Have you tried updating table filter to the latest version too?
If that doesn't fix it check the console for errors. There may be differences in the way tablefilter is configured in the new version.
